tennis-scheduler-web-app/src/components/Login.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from "react"
+import { Link } from 'react-router-dom'   
+
+export const Login = () => {
+    const initialValues = { email: "", password: ""};
+    const [formValues, setFormValues] = useState(initialValues);
+    const [formErrors, setFormErrors] = useState({});
+    const [isSubmit, setIsSubmit] = useState(false);
+
+    const loginSubmit = e => {
+        e.preventDefault(e)
+        setFormErrors(validation(formValues));
+        setIsSubmit(true);
+    }
+
+    const handleChange = e => {
+        const { name, value } = e.target;
+        setFormValues({ ...formValues, [name]: value });
```

Comment:
When working with multiple fields in one state you should use previousState e.g.
setFormValues((previousState) => ({...previousState, [name]: value}));
 There is always a possibility that you get older version of it e.g you get older version of password instead of current one -> React handles its states under the hood.